bbc/simorgh
# Bump rich-text-tranforms@1.1.1 to fix client side XML parsing bug

Resolves #4408

**Overall change:** Bumps `@bbc/psammead-rich-text-transforms@1.1.1`

---


## Testing notes
- `npm run start && npm run build` 
- visit http://localhost.bbc.com:7080/pidgin/tori-49450859
- click "Test MAP to MAP inline link" 
- Ensure that paragraphs that contain bold (blocks that have `candy_xml` mark up type) render as expected. This can also be checked by refreshing the page to get a server render and ensure no new content appears after the refresh. 

# Client side parsing bug prior to PR
![Client side parsing failing](https://user-images.githubusercontent.com/7791726/67566081-d0763c80-f71e-11e9-87b6-7e50ac87a691.gif)

# Client side render following this PR
![Client side parsing working](https://user-images.githubusercontent.com/7791726/67566502-d7517f00-f71f-11e9-8d6f-84872b67925f.gif)
